subprojects/core/src/main/java/org/gradle/caching/configuration/internal/DefaultBuildCacheConfiguration.java

```diff
@@ -56,13 +57,17 @@ public DirectoryBuildCache getLocal() {
 
     @Override
     public <T extends DirectoryBuildCache> T local(Class<T> type) {
-        DeprecationLogger.deprecateMethod("BuildCacheConfiguration.local(Class)").replaceWith("getLocal()").nagUser();
+        DeprecationLogger.deprecateMethod("BuildCacheConfiguration.local(Class)").replaceWith("getLocal()")
```

Comment:
I think upgrade guide 5 section "Local build cache is always a directory cache" is a better fit for this.